far/farlang.templ.m4

```diff
@@ -2251,12 +2251,12 @@ upd:"Highlight column separators"
 "Підсвічувати роздільники колонок"
 
 MConfigDoubleGlobalColumnSeparator
```

Comment:
Should I rename MConfigDoubleGlobalColumnSeparator to MConfigDoubleStripeSeparator? There is also DoubleGlobalColumnSeparator in config.cpp. Renaming of the latter is probably not advisable, because it will be a breaking change to config DB, right?

Reply:
It is ok to rename language ids accordingly.
Renaming db strings would be a breaking change, so yes, it is better to leave them as is.